src/stage/Stage1.ts

```diff
@@ -0,0 +1,17 @@
+import { StageDefinition } from './StageDefinition';
+
+export const stage1: StageDefinition = {
```

Comment:
what about `stage01` for the name and the file to order files ?

Reply:
Good idea 💡